src/Settings/SettingsValidator.php

```diff
@@ -89,7 +86,12 @@ private function has_valid_store_name() {
 	 * @return bool
 	 */
 	private function has_valid_email() {
-		return ! empty( filter_var( $this->settings->get_email_address(), FILTER_VALIDATE_EMAIL ) );
+		// Not required.
+		if( empty( $this->settings->get_email_address() ) ) {
+			return true;
+		}
```

Comment:
@bplv112 I am sure that you will have tested this code by now. Can you please help me make sense of this logic?

Having an email address string returns true? Shouldn't this return false?

Also, can you please add a space after `if`?

Reply:
@asharirfan Previously email address were required but now it is not. It's optional so if it's empty we don't validate it at all and it's the same for phone number.

Reply:
@bplv112 Alright